requests/adapters.py

```diff
@@ -59,7 +59,7 @@ def __init__(self):
         super(BaseAdapter, self).__init__()
 
     def send(self, request, stream=False, timeout=None, verify=True,
-             cert=None, proxies=None):
+             cert=None, proxies=None, proxies_kwargs=None):
```

Comment:
This is the probably the more controversial part of this change. Requests has considered its top level APIs more-or-less frozen since 2.0. Adding this is a deviation from that philosophy.

While we should continue to uphold that stance, I don't see another way for us to expose this urllib3 functionality. Given we have some legitimately broken use cases, I think a rare exception may be warranted.